Dockerfile-dev

```diff
@@ -0,0 +1,14 @@
+FROM python:3.6
+
+WORKDIR /usr/src/app
+COPY requirements/ /usr/src/app/requirements/
+
+RUN pip install --no-cache-dir --requirement requirements/production.txt
```

Comment:
Doesn't look like dev requirements ;)

Reply:
made consistent with Dockerfile and using `requirements/base.txt` now